var/spack/repos/builtin/packages/subread/package.py

```diff
@@ -13,10 +13,10 @@ class Subread(MakefilePackage):
     sequencing data."""
 
     homepage = "http://subread.sourceforge.net/"
```

Comment:
Please use the `https` protocol even here.

```suggestion
    homepage = "https://subread.sourceforge.net/"
```